tests/functional/test_runtime_parameters.py

```diff
@@ -216,6 +218,10 @@ def _test_custom_model_with_runtime_params(
             cmd += f"--runtime-params-file {runtime_param_values_stream.name}"
         else:
             env["RUNTIME_PARAMS_FILE"] = runtime_param_values_stream.name
+
+        if docker:
```

Comment:
is this only testable through functional tests?  if there's no way to unit test this, then it would be worthwhile to create one.